src/external-queries.ts

```diff
@@ -3,26 +3,26 @@ import * as exec from '@actions/exec';
 import * as fs from 'fs';
 import * as path from 'path';
 
-import * as configUtils from './config-utils';
 import * as util from './util';
 
-export async function checkoutExternalQueries(config: configUtils.Config) {
+/**
+ * Checkout a repository at the given ref, and return the directory of the checkout.
```

Comment:
```suggestion
 * Check out repository at the given ref, and return the directory of the checkout.
```

Removing the "a" makes it clearer how you're using the `repository` parameter.